code/datums/discord/webhook.dm

```diff
@@ -0,0 +1,154 @@
+#define HEX_COLOR_RED 16711680
+#define HEX_COLOR_GREEN 65280
+#define HEX_COLOR_BLUE 255
+#define HEX_COLOR_YELLOW 16776960
+
+var/list/global_webhooks = list()
+
+/proc/global_initialize_webhooks()
+	if (!establish_db_connection(dbcon))
+		var/file = return_file_text("config/webhooks.json")
+		if (file)
+			var/jsonData = json_decode(file)
+			if(!jsonData)
+				return 0
+			for(var/hook in jsonData)
+				if(!hook["url"] || !hook["tags"])
+					continue
+				var/datum/webhook/W = new(hook["url"], hook["tags"])
+				global_webhooks += W
+				if(hook["mention"])
+					W.mention = hook["mention"]
+			return 1
+		else
+			return 0
+	else
+		var/DBQuery/query = dbcon.NewQuery("SELECT url, tags, mention FROM ss13_webhooks")
+		query.Execute()
+
+		if (query.ErrorMsg())
+			return 0
```

Comment:
If there is an error with the query, log it.
Silently throwing away error messages makes debugging much harder if something is not working.